tpv/core/entities.py

```diff
@@ -578,6 +594,74 @@ def from_dict(loader, entity_dict):
             handler_tags=entity_dict.get('tags')
         )
 
+    def to_dict(self):
+        dest_dict = {
+            'id': self.id,
+            'abstract': self.abstract,
+            'runner': self.runner,
+            'destination_name_override': self.dest_name,
+            'cores': self.cores,
+            'mem': self.mem,
+            'gpus': self.gpus,
+            'min_cores': self.min_cores,
+            'min_mem': self.min_mem,
+            'min_gpus': self.min_gpus,
+            'max_cores': self.max_cores,
+            'max_mem': self.max_mem,
+            'max_gpus': self.max_gpus,
+            'min_accepted_cores': self.min_accepted_cores,
+            'min_accepted_mem': self.min_accepted_mem,
+            'min_accepted_gpus': self.min_accepted_gpus,
+            'max_accepted_cores': self.max_accepted_cores,
+            'max_accepted_mem': self.max_accepted_mem,
+            'max_accepted_gpus': self.max_accepted_gpus,
```

Comment:
This needs to be split I think, otherwise, the superclass method will be aware of its subclasses. The `min_accepted` related fields should go to the `Destination` class, and rules should go to the `EntityWithRules` class.